src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java

```diff
@@ -260,11 +260,14 @@ private Bundle processBundle(Bundle bundle) {
       }
 
       // generate random responses since not cancelling
-      switch (getRand(3)) {
+      switch (getRand(6)) {
       case 1:
         responseDisposition = "Granted";
```

Comment:
Having a comment her about the probability for each case would be nice.   I can intuit what you're doing, but stating it explicitly is better.

Reply:
Added a comment about the probability